### ISSUE TITLE
A confusing line of  code, which may have to be modified

### ISSUE THREAD

**Issue by shawnfd** [ChingHo97/FCOS-PyTorch-37.2AP#24]
Hi zhenghao,
    the train/eval mode of the fcos model is written in your code [path/model/fcos.py], however, in the line 26
'super().train(mode=True)', the model mode is fixed as training mode.
   That leads to some unexpected inference results when doing inference by set model to eval(). So, I suggest 
modify 'super().train(mode=True)' to 'super().train(mode=mode)'.
   Otherwise, the dropout/bn is not locked in the inference procedure. At the same time, the dropout/bn status
can be affected by the [path/model/config.py] configurations: freeze_stage1/freeze_bn

**Comment by zhenghao977** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
